src/utils/getSortedGoods.ts

```diff
@@ -0,0 +1,36 @@
+import { SortType } from '../types/SortType';
+
+interface SortingFilters {
+  sortField: SortType;
+  isReversed: boolean;
+}
+
+export function getSortedGoods(
```

Comment:
The function's name `getSortedGoods` is good as it starts with a verb, but it's not clear what sorting criteria it uses without looking at the implementation. Consider renaming it to something more descriptive, like `sortGoodsByCriteria` to improve code readability.